server/src/immich/controllers/asset.controller.ts

```diff
@@ -145,4 +147,16 @@ export class AssetController {
   ): Promise<AssetResponseDto> {
     return this.service.update(authUser, id, dto);
   }
+
+  @Post('stack')
+  @HttpCode(HttpStatus.OK)
+  updateStack(@AuthUser() authUser: AuthUserDto, @Body() dto: UpdateAssetStackDto): Promise<void> {
+    return this.service.updateStack(authUser, dto);
```

Comment:
Can this be merged/implemented with just `PUT /asset`. I believe this is bulk asset update endpoint and it seems perfect to handle bulk setting the stack parent on multiple assets.

Reply:
Removed `POST /asset/stack` asset and updated `PUT /asset` to handle stacking as well. Had to add one more entry `removeParent` to `AssetBulkUpdateDto` other than `stackParentId`  because the mobile app cannot send both undefined and null for `stackParentId` to handle parent removal.